README.md

```diff
@@ -184,7 +184,7 @@ If you want to contribute to this list (please do), send me a pull request.
 * [graphql-compose](https://github.com/graphql-compose/graphql-compose) - Tool which allows you to construct flexible graphql schema from different data sources via plugins.
 * [node-graphjoiner](https://github.com/mwilliamson/node-graphjoiner) - Create GraphQL APIs using joins, SQL or otherwise.
 * [Join Monster](https://github.com/acarl005/join-monster) - A GraphQL-to-SQL query execution layer for batch data fetching.
-* [graphql-factory](https://github.com/graphql-factory) - Create GraphQL types from JSON definitions
+* ~~[graphql-factory](https://github.com/graphql-factory) - Create GraphQL types from JSON definitions~~
```

Comment:
@alexander-mart Could you delete this line entirely instead of adding `~~` to it?